tutorials/Python/00_Introduction/Introduction_solutions.ipynb

```diff
@@ -35,22 +36,23 @@
     "8. Documentation\n",
     "9. Libraries\n",
     "10. Numpy\n",
-    "11. Analysing MD\n",
+    "11. OOP (Object Orientated Programming)\n",
```

Comment:
See above (this duplication of the solutions is not sustainable in the long run...)